pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -384,6 +384,7 @@ stdenv.mkDerivation {
       handle very large projects with speed and efficiency.
     '';
 
+    outputsToInstall = [ "out" "doc" "man" ];
```

Comment:
That does not work when not building the manual.

Reply:
Seems to have fixed.
How to reproduce a test on a local system?

Reply:
Not a good idea which does not require lots of RAM.